### PR TITLE
{lib}[foss/2023a] phono3py v2.7.0

### PR DESCRIPTION
**Pull request by jkuhl-uni**
Using
- #19460

, I'm adding phono3py in foss 2023a

### CODE REVIEW

**Comment by akesandgren** [comment]
@boegelbot Please test @ generoso

File: easybuild/easyconfigs/p/phono3py/phono3py-2.7.0-foss-2023a.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+name = 'phono3py'
+version = '2.7.0'
+
+homepage = 'http://phonopy.github.io/phono3py/'
+description = """A simulation package of phonon-phonon interaction related properties."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+source_urls = ['https://github.com/phonopy/phono3py/archive/refs/tags/']
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/phono3py/phono3py-2.7.0-foss-2023a.eb:11]: Use this instead
```
github_account = 'phonopy'
source_urls = [GITHUB_SOURCE]
```
And double check the checksum, the test build I just did complained about the checksum

**Review by akesandgren — CHANGES_REQUESTED** [review]

**Comment by akesandgren** [comment]
@boegelbot Please test @ generoso

**Review by akesandgren — APPROVED** [review]
LGTM

**Comment by jkuhl-uni** [comment]
Cool, thanks for helping me :)

**Comment by akesandgren** [comment]
Going in, thanks @jkuhl-uni!

**Pull request merged**